torch/nn/parallel/distributed.py

```diff
@@ -263,6 +320,8 @@ def __init__(self, module, device_ids=None,
                  bucket_cap_mb=25,
                  find_unused_parameters=False,
                  check_reduction=False):
+        # Dump DDP-relevant environment variables for debugging
+        _dump_DDP_relevant_env_vars()
```

Comment:
Shall we remove this from DDP ctor, as this info is probably only useful when debugging. If applications need to check env vars, they can explicitly call `_dump_DDP_relevant_env_vars`.